packages/css/js/Css_Js_Core.ml

```diff
@@ -645,6 +645,51 @@ let height x =
       | #Var.t as va -> Var.toString va
       | #Cascading.t as c -> Cascading.toString c )
 
+let textEmphasisStyle x =
+  D
+    ( {js|textEmphasisStyle|js},
+      match x with
+      | #TextEmphasisStyle.t as tes -> TextEmphasisStyle.toString tes
+      | `textEmphasisStyleXY (x, y) ->
+        (match x with
+        | #TextEmphasisStyle.FilledOrOpen.t as fo ->
+          TextEmphasisStyle.FilledOrOpen.toString fo
+        | #Var.t as va -> Var.toString va
+        | #Cascading.t as c ->
+          Cascading.toString c
+          ^ {js| |js}
+          ^
+          (match y with
+          | #TextEmphasisStyle.Shape.t as shape ->
+            TextEmphasisStyle.Shape.toString shape
+          | #Var.t as va -> Var.toString va
+          | #Cascading.t as c -> Cascading.toString c))
+      | #Var.t as va -> Var.toString va
+      | #Cascading.t as c -> Cascading.toString c )
+
+let textEmphasisPosition x =
+  D
+    ( {js|textEmphasisPosition|js},
+      match x with
+      | #TextEmphasisPosition.OverOrUnder.t as ou ->
+        TextEmphasisPosition.OverOrUnder.toString ou
+      | `textEmphasisPositionYX (y, x) ->
```

Comment:
I meant another function and the codegen renders one or the other

Reply:
I don't think I understand the need for another function, what would it do different?

Reply:
`let textEmphasisPosition`
`let textEmphasisPositions`

So the textEmphasisPosition actual doesn't need to handle the rare `(y, x)` case

Reply:
and what shall be the expression, `CssJs.textEmphasisPosition` or `CssJs.textEmphasisPositions`.
This is what I think you meant
```
let textEmphasisPosition x =
  D
    ( {js|textEmphasisPosition|js},
      match x with
      | #TextEmphasisPosition.OverOrUnder.t as ou -> ... )

let textEmphasisPositions value =
  D
    ( {js|textEmphasisPositions|js},
      match value with
      | x, y ->
        (match x with ... (match y with ...))
```
Is that so?